algorithms/hhl/hhl.ipynb

```diff
@@ -0,0 +1,1367 @@
+{
```

Comment:
<p>Actually, the scaling that ensures the same error is expected to be 4^s.</p><p>In practice, below we do not really use this scaling, but a much smaller one (1.8).</p><p> </p><p>You can write that the <code>R0</code> and <code>REPS_SCALING_FACTOR</code> below where chosen by trail an error. For other examples one would need to use other parameters, as these are Hamiltonian dependent. The relevant literature that discuss the errors of product formulas is <a href="https://journals.aps.org/prx/abstract/10.1103/PhysRevX.11.011020" target="_blank">https://journals.aps.org/prx/abstract/10.1103/PhysRevX.11.011020</a>   </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Classiq/classiq-library/pull/194/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='45'/>